content/en/xtc/loadtesting/170-monitor-lt.md

```diff
@@ -24,8 +24,23 @@ Steps that failed due to errors (like timeouts or failed authentication) will be
 
 ### Scenario Status
 
-The _Scenario Status_ tab only contains information while the test is actually running or finished. This is an overview of all test scenarios that are executed in this load test, containing scenario name and state and other useful information such as currently running users, average scenario runtime and especially the number of events and errors that occurred in this scenario:
+The _Scenario Status_ tab only contains information while the test is actually running or finished. After the [live metrics]({{< relref "#live-metrics" >}}) follows an overview of all test scenarios that are executed in this load test, containing scenario name and state and other useful information such as currently running users, average scenario runtime and especially the number of events and errors that occurred in this scenario:
 
 {{< image src="xtc/loadtest_scenarioStatus.png" >}}
 The scenario status view for a running load test.
-{{< /image >}}
\ No newline at end of file
+{{< /image >}}
+
+#### Live Metrics
+
+The graph on top of the scenario status table is updated live while the load test is running and displays information on the number of currently active users for each test scenario. Hovering over the graph displays a popup with the values for this point in time (number of total users in brackets).
+
+Since there might be a lot of information, you can **filter** in the dropdown for single or several test scenarios or just display total values. If you want more granularity in the metrics, selecting a higher **scaling factor** will make the chart taller. 
```

Comment:
```suggestion
Since there might be a lot of information, you can **filter** in the dropdown for single or several test scenarios or just display total values. If you want to see more details in the chart, selecting a higher **scaling factor** will make the chart taller. 
```